contracts/Forwarder.sol

```diff
@@ -75,18 +76,15 @@ contract Forwarder is ReentrancyGuard, ERC721TokenReceiver {
    * ERC721 standard callback function for when a ERC721 is transfered. The forwarder will send the nft
    * to the base wallet once the nft contract invokes this method after transfering the nft.
    *
-   * @param _operator The address of the nft contract
+   * @param _operator The address which called `safeTransferFrom` function
    * @param _from The address of the sender
    * @param _tokenId The token id of the nft
    * @param _data Additional data with no specified format, sent in call to `_to`
    */
   function onERC721Received(address _operator, address _from, uint256 _tokenId, bytes memory _data) public virtual override nonReentrant returns (bytes4) {
     if(autoFlush721){
-      ERC721 instance = ERC721(_operator);
-
+      ERC721 instance = ERC721(msg.sender);
```

Comment:
I have changed _operator to sender because as per my understanding _operator would the address which called safeTransferFrom function.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/ERC721.sol#L395

Reply:
Good find!